modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.tsx

```diff
@@ -174,7 +174,10 @@ const Card = forwardRef<HTMLDivElement, any>(
 							});
 
 							onSelect?.({selectedItems: [item]});
-							event.preventDefault();
+
+							if (event.target.tagName.toLowerCase() !== 'a') {
```

Comment:
nit: why are we checking lowercase? Isn't the spec of tagName always uppercase? So we can just remove the toLowerCase and use `'A'`